packages/ndla-ui/src/CopyParagraphButton/CopyParagraphButton.tsx

```diff
@@ -79,9 +80,7 @@ const CopyParagraphButton = ({ title }: Props) => {
           <Link title={''} />
         </Tooltip>
       </IconButton>
-      <h2 id={sanitizedTitle} tabIndex={0}>
-        {title}
-      </h2>
+      <h2 id={sanitizedTitle} tabIndex={0} dangerouslySetInnerHTML={{ __html: content || '' }}></h2>
```

Comment:
Burde dette heller vore med title?
```suggestion
      <h2 id={sanitizedTitle} tabIndex={0} dangerouslySetInnerHTML={{ __html: content || title }}></h2>
```

Reply:
Ser for meg at content med hensikt kan være `null` eller `''` i noen tilfeller. Tenker det er best å ikke fallbacke til en annen variabel.